docs/aztec/how-it-works/l1-l2-messaging.md

```diff
@@ -47,10 +47,6 @@ Because everything is unilateral and async, the application developer have to ex
 
 A "portal" refers to the part of an application residing on L1, which is associated with a particular L2 address (the confidential part of the application). The link between them is established explicitly to reduce access control complexity. On public chains, access control information such as a whitelist in a mapping or similar data structure can simply be placed in public storage. However, this is not feasible for contracts in Aztec. Recall that public storage can only be accessed (up to date) by public functions which are called AFTER the private functions. This implies that access control values in public storage only work for public functions. One possible workaround is to store them in private data, but this is not always practical for generic token bridges and other similar use cases where the values must be publicly known to ensure that the system remains operational. Instead, we chose to use a hard link between the portal and the L2 address.
 
-:::info
-Note, that we at no point require the "portal" to be a contract, it could be an EOA on L1.
```

Comment:
This is referring to an EOA on L1 Ethereum, so the concept of an EOA is still relevant.